Makefile

```diff
@@ -45,33 +48,41 @@ prereqs: $(BINDIR)	## Download required utilities
 git-clone-repos:	## Clone repositories from submariner-io
 git-clone-repos:	clone-admiral clone-cloud-prepare clone-lighthouse clone-submariner
 git-clone-repos:	clone-submariner-operator clone-shipyard
+	@echo -- $@ --
 
 clone-%: %/.git
+	@echo -- $@ --
```

Comment:
I think the `echo -n` was added solely for having a step in the target. If you have the `echo -- $@ --` in there, the other step is not needed